mod_reforged/hooks/skills/effects/bleeding_effect.nut

```diff
@@ -86,11 +86,10 @@
 			// so that the damage does not trigger morale checks
```

Comment:
The comment above this line (`::Const.Morale.OnHitMinDamage`) is no longer accurate